src/org/labkey/test/tests/UserTableCustomFieldUpdateTest.java

```diff
@@ -0,0 +1,112 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.remoteapi.query.UpdateRowsCommand;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.components.DomainDesignerPage;
+import org.labkey.test.components.domain.DomainFormPanel;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.util.DataRegionTable;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+@Category({Daily.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 3)
+public class UserTableCustomFieldUpdateTest extends BaseWebDriverTest
+{
+    private static final String CUSTOM_FIELD1 = "CustomField1";
+    private static final String CUSTOM_FIELD2 = "CustomField2";
+    private static final String TEST_USER = "testcustomfieldupdate@test.com";
+
+    @Override
+    protected @Nullable String getProjectName()
+    {
+        return getClass().getSimpleName() + " Project";
+    }
```

Comment:
This can/should return `null` if the test doesn't actually create a project.
```suggestion
    protected @Nullable String getProjectName()
    {
        return null;
    }
```